config/env_sample.json

```diff
@@ -168,22 +168,20 @@
         "videos": {"types": ["leccap"], "icon": "fas fa-video fa-lg"}
     },
     "/* Institute specific urls add or remove to your needs, will be used for Resources view": "*/",
-    "RESOURCE_URLS": {
-        "canvas": {"prefix": "https://example.instructure.com/files/", "postfix": "/download?download_frd=1"},
-        "leccap": {"prefix": "https://leccap.engin.umich.edu/leccap/viewer/r/", "postfix": ""}
-    },
     "/*cron queries to pull the data from various caliper data sources replace if not relevant with your institution": "*/",
     "RESOURCE_ACCESS_CONFIG": {
         "canvas_data": {
```

Comment:
@jonespm, you said this should be "canvas", correct?

Reply:
Yeah right, thanks!

Reply:
```suggestion
        "canvas": {
```